FRTestHost/FRTestHost/SharedTestFiles/TestConfig/Config-live-01.json

```diff
@@ -1 +1,52 @@
-
+{
+    "username": "sdkuser",
+    "password": "password",
```

Comment:
Is it ok to expose the password?

Reply:
uh, oh! of course not! 
I fixed that by putting the password in github actions secret...